### PR TITLE
Update hiding-tabbar-in-screens.md

### PR DESCRIPTION
**Pull request by Qantt**
* Text talks about Notifications screen, however notifications screen is not currently present in snippets => added Notifications screen to snippets

# READ ME PLEASE!

### TL;DR: Make sure to add your changes to versioned docs

Thanks for opening a PR!

The docs cover several versions of `react-navigation`, and in some cases there are several files (for version 1, version 2 and etc.) that all describe a single page of the docs (eg. "Getting Started").

Please make sure that the edit you're making in `docs/file-you-edited.md` is also included in the file for the correct version, eg. `/versioned_docs/version-3.x/file-you-edited.md` for version 3. If such file doesn't exist, please create it. :+1:

### CODE REVIEW

File: versioned_docs/version-5.x/hiding-tabbar-in-screens.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ function App() {
     <Tab.Navigator>
       <Tab.Screen name="Home" component={HomeStack} />
       <Tab.Screen name="Feed" component={Feed} />
-      <Tab.Screen name="Settings" component={Settings} />
+      <Tab.Screen name="Notifications" component={Notifications} />
```
Comment (jarvisluong) [inline versioned_docs/version-5.x/hiding-tabbar-in-screens.md:25]:
```suggestion
      <Tab.Screen name="Settings" component={Settings} />
      <Tab.Screen name="Notifications" component={Notifications} />
```

Settings screen is still mentioned in the doc 😃

File: versioned_docs/version-5.x/hiding-tabbar-in-screens.md [hunk 2]
```diff
@@ -38,7 +38,7 @@ function HomeTabs() {
     <Tab.Navigator>
       <Tab.Screen name="Home" component={Home} />
       <Tab.Screen name="Feed" component={Feed} />
-      <Tab.Screen name="Settings" component={Settings} />
+      <Tab.Screen name="Notifications" component={Notifications} />
```
Comment (jarvisluong) [inline versioned_docs/version-5.x/hiding-tabbar-in-screens.md:41]:
```suggestion
      <Tab.Screen name="Settings" component={Settings} />
      <Tab.Screen name="Notifications" component={Notifications} />
```

Settings screen is still mentioned in the doc 😃

**Comment by jarvisluong** [comment]
After reading the whole page section, I agree that we should update the code example to be more close to the text description. Could you make more detailed updates?

File: versioned_docs/version-5.x/hiding-tabbar-in-screens.md (hunk 2 above)
Comment (Qantt) [reply versioned_docs/version-5.x/hiding-tabbar-in-screens.md:41]: Settings screen is still in App component, so I don't think it should be duplicated here.

File: versioned_docs/version-5.x/hiding-tabbar-in-screens.md (hunk 1 above)
Comment (Qantt) [reply versioned_docs/version-5.x/hiding-tabbar-in-screens.md:25]: Settings screen is still in HomeStack component, so I don't think it should be duplicated here.

Comment (jarvisluong) [reply versioned_docs/version-5.x/hiding-tabbar-in-screens.md:25]: Ah yes, you are right!

**Pull request merged**